azure-servicebus/src/main/java/com/microsoft/azure/servicebus/ISubscriptionClient.java

```diff
@@ -15,6 +16,8 @@
 	public CompletableFuture<Void> addRuleAsync(String ruleName, Filter filter);
 	public CompletableFuture<Void> removeRuleAsync(String ruleName);
 	public void removeRule(String ruleName) throws InterruptedException, ServiceBusException;
+	public List<RuleDescription> getRules() throws ServiceBusException, InterruptedException;
```

Comment:
>List<RuleDescription> [](start = 8, length = 21)

Change this to Collection<RuleDescription>. That is what we have been using for all methods which return multiple objects.